src/de/mod.rs

```diff
@@ -216,6 +216,10 @@ impl<'a> QsDeserializer<'a> {
     fn with_config(config: &Config, input: &'a [u8]) -> Result<Self> {
```

Comment:
Can you make `with_config` public too? I'm using strict=false so the default config won't work for me. Thanks!

Note: I'm just another user that also wants this, I don't have merge rights.
```suggestion
    pub fn with_config(config: &Config, input: &'a [u8]) -> Result<Self> {
```